### PR TITLE
Disable offline mode by default

### PR DESCRIPTION
**Pull request by superm1**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [x] Feature
- [ ] Documentation

### CODE REVIEW

File: data/bash-completion/fwupdmgr [hunk 1]
```diff
@@ -56,7 +56,6 @@ _fwupdmgr_cmd_list=(
 
 _fwupdmgr_opts=(
 	'--verbose'
-	'--offline'
```
Comment (hughsie) [inline data/bash-completion/fwupdmgr:59]: Shouldn't this be dependant on whether the option is set? Can we even do that?

Comment (superm1) [reply data/bash-completion/fwupdmgr:59]: fixed

File: plugins/genesys/fu-genesys-usbhub-device.c [hunk 2]
```diff
@@ -1878,11 +1878,9 @@ fu_genesys_usbhub_device_prepare(FuDevice *device,
...
 
 	fu_progress_set_id(progress, G_STRLOC);
-	fu_progress_add_step(progress, FWUPD_STATUS_SCHEDULING, 5, "detach");
-	for (guint i = 0; i < fw_types->len; i++) {
-		fu_progress_add_step(progress, FWUPD_STATUS_SCHEDULING, 100, NULL);
-		fu_progress_add_step(progress, FWUPD_STATUS_SCHEDULING, 100, NULL);
-	}
+	fu_progress_add_step(progress, FWUPD_STATUS_LOADING, 5, "detach");
+	for (guint i = 0; i < fw_types->len; i++)
+		fu_progress_add_step(progress, FWUPD_STATUS_LOADING, 100, NULL);
```
Comment (hughsie) [inline plugins/genesys/fu-genesys-usbhub-device.c:1883]: I think you need two of these -- there are 2 of `fu_progress_step_done(progress)` in the `for (guint i = 0; i < fw_types->len; i++) {` loop.

Comment (superm1) [reply plugins/genesys/fu-genesys-usbhub-device.c:1884]: fixed

**Review by hughsie — APPROVED** [review]
Thanks!

**Pull request merged**